packages/smarthr-ui/src/components/Pagination/PaginationControllerItemButton.tsx

```diff
@@ -31,19 +33,34 @@ export const PaginationControllerItemButton: FC<Props> = ({
   disabled,
   double,
   targetPage,
+  hrefTemplate,
+  linkAs,
 }) => {
   const { Icon, alt } = ICON_MAPPER[direction][double ? 'double' : 'single']
 
+  const { Component, attrs } = useMemo(() => {
+    if (hrefTemplate) {
+      return {
+        Component: AnchorButton,
+        attrs: {
+          href: disabled ? undefined : hrefTemplate(targetPage),
+          elementAs: linkAs,
```

Comment:
`href`が`undefined`だとnext/linkがエラーになってしまうので
`disabled`の場合は通常のaタグにするようにしたいです
```suggestion
          elementAs: disabled ? undefined : linkAs
```